tests_e2e/test_todo_app.py

```diff
@@ -1,33 +1,35 @@
 import os
 from time import sleep
+import uuid
 from dotenv import load_dotenv, find_dotenv
 from threading import Thread
+import pymongo
 import pytest
 from selenium import webdriver
 from selenium.webdriver.common.keys import Keys
 from selenium.webdriver.common.by import By
 from selenium.webdriver.firefox.options import Options
-from tests_e2e.trello_board_helpers import create_trello_board, delete_trello_board
 from todo_app import app
 
 @pytest.fixture(scope='module')
 def driver():
     opts = Options()
     opts.headless = True
     with webdriver.Firefox(options=opts) as driver:
-         # Wait up to 2 seconds when looking for an element
-        driver.implicitly_wait(2)
+         # Wait up to 10 seconds when looking for an element
+        driver.implicitly_wait(10)
 
         yield driver
 
 @pytest.fixture(scope='module')
-def app_with_temp_board():
+def app_with_temp_db():
     file_path = find_dotenv('.env')
     load_dotenv(file_path, override=True)
 
-    # Create the new board & update the board id environment variable
-    board_id = create_trello_board()
-    os.environ['TRELLO_BOARD_ID'] = board_id
+    # Update the database name to use a random test database so concurrent tests do not interfere
+    os.environ['MONGO_DATABASE_NAME'] = 'test_database_'+str(uuid.uuid1())

```

Comment:
Fun bug; I'm using a MongoDB rather than a CosmosDB for testing, and it threw an error here saying:
> Database name test_database_b1c20e78-2913-11ed-8499-9cb6d0e44c2c is too long

Not an issue since we're using Cosmos, but could be one to just reduce the size of this!